scaffold-example.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
+
+const path = process.argv[2];
+
+if (! path || path === '') {
+  console.log('You must provide a path for the code example');
+  console.log('> node scaffold-example.js <path> <example name>');
+  process.exit(1);
+}
+
+if (! fs.existsSync(path)) {
+  console.log(`Path ${path} does not exists`);
+  process.exit(1);
+}
+
+const exampleName = process.argv[3];
+
+if (! exampleName || exampleName === '') {
+  console.log('You must provide an example name for the code example');
+  console.log('> node scaffold-example.js <path> <example name>');
+  process.exit(1);
+}
+
+
+
+
+['js', 'go', 'cpp', 'java'].forEach(language => {
```

Comment:
nitpicking: I would rather use a  ̀for .. of` loop

Reply:
For my part I prefer functional style (since this is not a critical section of code)